### ISSUE TITLE
Indicate dynamic change of watch variables to screen reader users

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#221604]
Sighted users have access to dynamic variable state, screen reader users do not. We should announce watch dynamic variable changes.

Consider if just variable name or variable name + value should be announced

**Comment by meganrogge** [comment]
@jooyoungseo and @rpereze030 do we think alert or status would be better here? I'll test each.

**Comment by meganrogge** [comment]
In testing, the screen reader is not announcing the alert / status update. Need to maybe rework this.
